spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,37 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts controller', type: :request do
+  describe 'GET /index' do
+    before :each do
+      get '/users/1/posts'
+    end
+    it 'renders template' do
+      expect(response).to render_template(:index)
+    end
+
+    it 'correct status' do
+      expect(response.status).to eq(200)
+    end
+
+    it 'body includes correct text' do
+      expect(response.body).to include('display list of posts')
+    end
+  end
+
+  describe 'GET /show' do
+    before :each do
+      get '/users/1/posts/2'
+    end
+    it 'renders template' do
+      expect(response).to render_template(:show)
+    end
+
+    it 'correct status' do
+      expect(response.status).to eq(200)
+    end
+
+    it 'body includes correct text' do
+      expect(response.body).to include('display certain user posts ')
+    end
```

Comment:
- [ ] <b>REQUIRED:</b> This test case is also failing, Kindly try to fix the issue.
![Screen Shot 2022-03-25 at 02 13 10](https://user-images.githubusercontent.com/45217364/160030080-eec44c52-011c-4dc5-9c07-6118292a1e66.png)